BackofficeBundle/Resources/public/coffee/backboneRouter.coffee

```diff
@@ -21,8 +21,22 @@ OrchestraBORouter = Backbone.Router.extend(
   initDisplayRouteChanges: (selector) ->
     selector = if selector == undefined then '[href="#' + Backbone.history.fragment + '"]' else selector
     $('nav li.active').removeClass 'active'
-    $('nav li:has(a' + selector + ')').addClass 'active'
-    openMenu menu_speed, openedSign
+    lis = $('nav li:has(a' + selector + ')')
+    lis.addClass 'active'
+    lis.each ->
+      ul = $('ul', this)
+      link = $(this).children('a').first()
+      if link.data('subtree')
+        $.ajax
+          url: link.data('subtree')
+          type: "GET"
+          async: false
```

Comment:
- Some menus use ajax, some others not. openMenu should be duplicate in case of using callback.
- In case of multiple tree ajax calls with callbacks, we could'nt send once openMenu.

For these reasons, async is here safe and make this functionality more flexible.